packages/docs/src/sidebar.mdx

```diff
@@ -2,15 +2,15 @@
   - [Getting Started with Gatsby](/getting-started/gatsby)
 - [Theming](/theming)
 - [The `sx` Prop](/sx-prop)
-- [Styling MDX](/styling-mdx)
 - [Color Modes](/color-modes)
-- [Themed](/themed)
-- [MDX Components](/mdx-components)
 - [Hooks](/hooks)
 - [API](/api)
 - [Theme Specification](/theme-spec)
 - [Demo](/demo)
 - [Resources](/resources)
+- [Styling MDX](/styling-mdx)

```

Comment:
I'm a supporter of this. Might think about grouping into an MDX category in the future, since the sidebar is so crowded, but this is fine for now